cmd/collectors/zapi/plugins/volume/volume.go

```diff
@@ -118,6 +118,12 @@ func (v *Volume) updateVolumeLabels(data *matrix.Matrix, volumeCloneMap map[stri
 			continue
 		}
 
+		// The ZAPI does include node root and temp volumes. Harvest will exclude them by not exporting them.
```

Comment:
ZAPI includes node root and temp volumes, while REST does not. To make ZAPI and REST consistent, Harvest will exclude the node root and temp volumes by not exporting them.

Reply:
added